wetterdienst/additionals/geo_location.py

```diff
@@ -131,3 +132,40 @@ def _derive_nearest_neighbours(latitudes_stations: np.array,
     return distance_tree.query(
         coordinates.get_coordinates_in_radians(),
         k=num_stations_nearby)
+
+
+def stations_to_geojson(df: pd.DataFrame):
```

Comment:
The return type is missing here

Reply:
I've added `dict` to describe the return type. Thanks again!